packages/tuono-router/src/components/NotFound.tsx

```diff
@@ -15,10 +15,14 @@ export function NotFound(): JSX.Element {
     return <RouteMatch route={custom404Route} serverInitialData={{}} />
   }
 
+  const RootLayout = router.routesById['__root__']?.component
+
+  if (!RootLayout) return null
+
   return (
-    <>
+    <RootLayout data={{}} isLoading={false}>
```

Comment:
Since there is no data, why not passing `null` instead of `{}`?

```suggestion
    <RootLayout data={null} isLoading={false}>
```